bigchaindb_driver/transport.py

```diff
@@ -0,0 +1,23 @@
+from .connection import Connection
+from .pool import Pool
```

Comment:
What's the status quo on relative imports? Are python3's relative imports OK to use?